actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -799,6 +805,12 @@ def init_with(coder) # :nodoc:
       end
     end
 
+    def resolve_nested_attributes # :nodoc:
```

Comment:
I would consider naming this `#each_nested_attribute`, and making it protected instead of public.